.github/workflows/deploy.yml

```
@@ -215,6 +215,11 @@ jobs:
     needs: [build]
     if: ${{ inputs.deploy_infrastructure && (inputs.environment || 'production') == 'production' }}
     environment: ${{ inputs.environment || 'production' }}
+    # Queue production state mutations in dispatch order and never interrupt one.
+    concurrency:
+      group: hov-production-terraform-state
+      cancel-in-progress: false
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)